opennms-dao/src/main/java/org/opennms/netmgt/dao/hibernate/AbstractDaoHibernate.java

```diff
@@ -433,4 +433,10 @@ private void logExtraSaveOrUpdateExceptionInformation(final T entity, final Data
             cause = cause.getCause();
         }
     }
+
+    @Override
+    public void deleteAll() {
+        String sql = String.format("delete from %s entity", this.m_entityClass.getSimpleName());
```

Comment:
The Usage of `this` and `m_` variables is kinda contrary.
I'd opt for use `m_entityClass` instead.

Reply:
obsolete, we use the GenericPersistenceAccessor